ew/static/weapons.py

```diff
@@ -1506,8 +1505,8 @@ def wef_harpoon(ctn = None):
         str_equip="You equip the model 397 hunting rifle.",
         str_name="model 397 hunting rifle",
         str_weapon="a model 397 hunting rifle",
-        str_weaponmaster_self="You are a rank {rank} gentleman of the model 397 hunting rifle.",
```

Comment:
lol, did you just find the whole 'gentleman' thing cringe? there are other weapons that use unique titles. i could take it or leave it, just curious on your reasoning

Reply:
No, it just doesnt make sense given that it isnt a hunting rifle anymore in terms of stats